src/client/inputhandler.h

```diff
@@ -285,6 +288,33 @@ class RealInputHandler : public InputHandler
 	{
 		return m_receiver->WasKeyReleased(keycache.key[k]) || joystick.wasKeyReleased(k);
 	}
+	virtual float getMovementSpeed() {
+		if (m_receiver->IsKeyDown(keycache.key[KeyType::FORWARD]) ||
+		    m_receiver->IsKeyDown(keycache.key[KeyType::BACKWARD]) ||
+		    m_receiver->IsKeyDown(keycache.key[KeyType::RIGHT]) ||
+		    m_receiver->IsKeyDown(keycache.key[KeyType::LEFT]))
```

Comment:
Please use to tabulator for continued condition lines.

Reply:
fixed with https://github.com/minetest/minetest/pull/11262/commits/537aca607c5f2164834c7b797c8afedbe0533f92